src/core/parser.ml

```diff
@@ -491,6 +491,12 @@ let term = term PFunc
 let parser rule =
   | l:term "↪" r:term -> Pos.in_pos _loc (l, r)
 
+(** [inductive] is a parser for a single inductive type. *)
+let parser inductive =
+  |     i:ident t:{":" term} "≔"
+    c:{ "|" ilist:ident ":" tlist:term }*
```

Comment:
ilist and tlist seem useless here

Reply:
and why not simply writing `i:ident ":" t:term ...` ?

Reply:
I have a Earley's error if I delete `ilist` or/and `tlist` and I don't konw why.
OK for the second remark.

Reply:
Do you also get an error if you replace ilist and tlist by _ ?

Reply:
Yes, it's strange.